Library/Formula/v8.rb

```diff
@@ -83,8 +93,8 @@ def install
     (buildpath/"tools/swarming_client").install resource("swarming_client")
 
     system "make", "native", "library=shared", "snapshot=on",
-                   "console=readline", "i18nsupport=off",
-                   "strictaliasing=off"
+                   "console=readline", i18nsupport,
```

Comment:
This changes uses icu4c by default, if you mean that. `i18nsupport` is set on/off based on `if build.with? "icu4c"` condition. If one will want to install without icu, thatn `--without-icu4c` should be specified. I build with ICU for consistency with how it built on debian/ubuntu and how v8 built in chrome.